DfciPkg/SettingsManager/SettingsManagerProvider.c

```diff
@@ -332,11 +332,10 @@ SetProviderValueFromAscii (
   return Status;
 }
 
-#define ENABLED_STRING_SIZE                (9)
-#define ASSET_TAG_STRING_MAX_SIZE          (22)
+#define ENABLED_STRING_SIZE                (13)
 #define SECURE_BOOT_ENUM_STRING_SIZE       (20)
 #define SYSTEM_PASSWORD_STATE_STRING_SIZE  (30)
-#define USB_PORT_STATE_STRING_SIZE         (20)

```

Comment:
We are enforcing changes to have as issue.  Please create an issue describing this bug.

Reply:
Here's the issue: https://github.com/microsoft/mu_feature_dfci/issues/41

Reply:
I added it to the commit message, so it links the PR to the issue. If you do that in the future, it will create a link between the two and automatically close the issue when the PR completes.